src/content/Settings/WorklistActions/SyncSavedSchedules/SyncSavedSchedules.tsx

```diff
@@ -0,0 +1,98 @@
+import { useContext, useState } from "react"
+import { ISectionData } from "../../../App/App.types"
+import {
+  getAllSavedScheduleIDs,
+  savedScheduleTerm,
+} from "./syncSavedSchedulesHelper"
+import { ModalDispatchContext, ModalPreset } from "../../../ModalLayer"
+import { addCoursesToSavedSchedule } from "../../../../backends/workday/savedSchedulesApi"
+import "./SyncSavedSchedules.css"
+import QuestionIcon from "../../../Icons/QuestionIcon"
+import { filterSections } from "../../../utils"
+
+interface IProps {
+  sections: ISectionData[]
+}
+const SyncSavedSchedules = (props: IProps) => {
+  const dispatchModal = useContext(ModalDispatchContext)
+  const [isOpen, setIsOpen] = useState(false)
+  const [selectedWorklist, setSelectedWorklist] = useState<number>(0)
+
+  const dispatchInstructions = () => {
+    dispatchModal({
+      preset: ModalPreset.SyncInstructions,
+    })
+  }
+
+  const beginSync = async () => {
+    const scheduleID = getAllSavedScheduleIDs()
+    if (scheduleID) {
+      const possibleErrors = await addCoursesToSavedSchedule(
+        filterSections(props.sections, selectedWorklist, savedScheduleTerm()),
+        scheduleID[0]
+      )
+      if (possibleErrors) {
+        dispatchModal({
+          preset: ModalPreset.SyncErrors,
+          additionalData: possibleErrors,
+        })
+      } else {
+        dispatchModal({
+          preset: ModalPreset.SyncConfirm,
+        })
+      }
+    } else {
+      alert(
+        `No saved schedule was detected. The most likely issue is that you are not on the "View My Saved Schedules" page. Please head to that page and try again.`
+      )
+      return
+    }
+  }
+
+  const handleClick = () => {
+    setIsOpen(!isOpen)
+  }
+
+  return (
+    <div className="sync-drawer">
+      <div className="title" onClick={handleClick}>
+        Sync To Saved Schedules {isOpen ? "\u25B2" : "\u25BC"}
+      </div>
+      {isOpen && (
+        <div className="drawer-content">
+          <div className="instructions-container">
+            <div className="instructions-prompt">Instructions</div>
+            <div
+              className="instructions-info-button"
+              onClick={dispatchInstructions}
+            >
+              <QuestionIcon size={16} />
+            </div>
+          </div>
+          <div className="worklist-container">
+            <div>Worklist: </div>
+            {[0, 1, 2, 3].map((worklist) => (
```

Comment:
can you not use `TabBar` here? would avoid the duplication of styles as well

Reply:
I did not know about that component lol. will try that